sysutils/node_exporter/pkg-descr

```diff
@@ -3,6 +3,9 @@ written in Go with pluggable metric collectors.
 
 WWW: https://github.com/prometheus/node_exporter
 
+***Please note that this needs to be enabled via Services: Prometheus Exporter.
```

Comment:
Looks like you want to add a plugin documentation instead? That would be a good place to add this kind of information about how to set it up.

Reply:
Considering I spent the afternoon trying to figure out how to do this, and the main place I looked was the plugin file I would say that as a starter for 10 this would be a good 0lace to start.  Beyond that some documentation would be good, but I think either the plugin or the service description needs changing to be consistent as it isn't immediately obvious that node-exporter relates to a service description of prometheus at all.  Just my 2p tho.